electron/src/runtime/lifecycle.ts

```diff
@@ -69,15 +69,17 @@ export const getWebViewId = (contents: WebContents): string | undefined => {
 // Using exit instead of quit for the time being
 // see: https://github.com/electron/electron/issues/8862#issuecomment-294303518
 export const quit = (): void => {
-  logger.info('Initiating app quit ...');
+  logger.info('Initiating app quit...');
   settings.persistToFile();
+  logger.info('Clear cache...');
+  session.defaultSession.clearCache().catch(() => {});
```

Comment:
```suggestion
  session.defaultSession.clearCache().catch(() => {});
```

There should be at least an error log when catching. Just swallowing the error is not a good practice.